catalyst/contrib/dl/callbacks/tests/test_cmc_metric_callback.py

```diff
@@ -0,0 +1,47 @@
+import torch
```

Comment:
I suggest you to use more `pytest` way for doing testing:
let's replace all of these functions in the test script with 2 function:
1. List of inputs and expected targets (don't forget about `pytest.fixture` decorator)
2. Single function which passes inputs to tested function and compares it with expected target

It will allow us to avoid copy pasting the code and it will be easier to add new data points for testing.

You can find examples in tests for triplets samplers

Reply:
And could you also add more difficult data points and/or automatic tests?

Reply:
I also think, that we need at least 1 end-to-end test with dummy hardcoded datapoint. 
This test should include creating of Callback instance, few batch updating steps and final score calculation.